static/src/javascripts/bootstraps/enhanced/notifications.js

```diff
@@ -43,37 +48,65 @@ define([
             return modules.getReg().then(function (reg) {return reg.pushManager.getSubscription();});
         },
 
-        configureSubscribeTemplate: function () {
-            var subscribed = modules.checkSubscriptions(),
-                hasNoSubscriptions = modules.subscriptionsEmpty(),
+        init: function() {modules.configureSubscribeButton();
+            if(!modules.hasSubscribed() && !modules.hasDismissedExplainer()) {
+                modules.showExplainer();
+            }
+        },
+
+        configureSubscribeButton: function () {
+            var $follow = bonzo(qwery('.js-live-notifications')[0]),
```

Comment:
We're already importing `$` here so you can just use `$('.js-live-notifications')`. You don't need the `[0] ` as `.html` is a bonzo method and you're already using `[0]` in the `.one` method already.